codalab/worker/dependency_manager.py

```diff
@@ -53,6 +98,9 @@ class DependencyManager(StateTransitioner, BaseDependencyManager):
     # the data format of how we store this)
     MAX_SERIALIZED_LEN = 60000
 
+    # If it has been this long since a worker has downloaded anything, another worker will take over downloading.
+    DEPENDENCY_DOWNLOAD_TIMEOUT_SECONDS = 30 * 60
```

Comment:
Only 5 mins is necessary

```suggestion
    DEPENDENCY_DOWNLOAD_TIMEOUT_SECONDS = 5 * 60
```